test/test.go

```diff
@@ -505,3 +505,15 @@ func (h *Helper) GetCommit(repo string) string {
 	}
 	return strings.TrimSpace(string(out))
 }
+
+func (h *Helper) GetLatestTag(repo string) string {
+	repoPath := h.Path("pkg/dep/sources/https---" + strings.Replace(repo, "/", "-", -1))
```

Comment:
Yeah, I didn't see that until after I'd put this up. Just tried that approach with the repo you mentioned, but since it actually depends on logrus as well that still ends up breaking the test. Looking into using the `gps.SourceManager` stuff now